src/wms/src/test/java/org/geoserver/wms/map/TinyRasterBoundingBoxTest.java

```diff
@@ -161,7 +161,7 @@ private Rectangle produceMap(double minX, double maxX, double minY, double maxY)
                         new ReferencedEnvelope(minX, maxX, minY, maxY, DefaultGeographicCRS.WGS84));
         RenderedImageMapOutputFormat rasterMapProducer = new RenderedImageMapOutputFormat(getWMS());
         RenderedImageMap imageMap = rasterMapProducer.produceMap(this.map);
-        this.op = (RenderedOp) imageMap.getImage();
+        this.op = (RenderedOp) ((RenderedImageTimeDecorator) imageMap.getImage()).getDelegate();
```

Comment:
Is this actually needed? The decorator should be transparent, a call to getImage() should generate the output just like calling directly its delegate. Same goes for the other casts in tests.